daemon/daemon.go

```diff
@@ -385,6 +385,17 @@ func (d *Daemon) initMaps() error {
 		return nil
 	}
 
+	// Delete old proxymaps if left over from an upgrade.
+	// TODO: Remove this code when Cilium 1.6 is the oldest supported release
+	for _, name := range []string{"cilium_proxy4", "cilium_proxy6"} {
+		path := bpf.MapPath(name)
+		if _, err := os.Stat(path); err == nil {
+			if err = os.RemoveAll(path); err == nil {
```

Comment:
Does this have any impact on upgrading then deciding to downgrade? Or same as any L7 impact (process is restarted so connections are reset)?

Reply:
Correct, no more impact than normal restart, as the L7 proxies are restarted with the cilium-agent.